pkg/reporting/nodes.go

```diff
@@ -45,12 +45,29 @@ func (nri *NodeReportItem) Array() []string {
 	for _, v := range nri.CookbookVersions {
 		cookbooks = append(cookbooks, v.String())
 	}
-	return []string{nri.Name,
-		nri.ChefVersion,
-		nri.OS,
-		nri.OSVersion,
-		strings.Join(cookbooks, " "),
+	var chefVersion string
+	if nri.ChefVersion == "" {
+		chefVersion = "-"
```

Comment:
I feel this formatting belongs in the rendering of the table, not in the report generation.